### PR TITLE
Updating gemspec redcarpet version to 3.4.0 and pedantic style changes

### PR DESCRIPTION
**Pull request by tomlockwood**
Hi Adam,

This is my first commit to a random open source project on github, I'm very excited and probably committing many faux pas.  Sorry!

I want to use your gem, but I get errors in my audit tool saying the 2.0 version of redcarpet has security vulns in it.  Can we possibly update it?

Thanks,
Tom

### CODE REVIEW

File: redcarpet-confluence.gemspec [hunk 1]
```diff
@@ -1,19 +1,27 @@
 Gem::Specification.new do |gem|
   gem.name          = 'redcarpet-confluence'
-  gem.version       = '1.1.0'
+  gem.version       = '1.2.0'
```
Comment (adamstegman) [inline redcarpet-confluence.gemspec]: No need to update the version number—1.1.0 is unreleased.

File: redcarpet-confluence.gemspec [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+                        'Rakefile',
+                        'redcarpet-confluence.gemspec'
+                      ]
+  gem.test_files    = Dir[
+                        'spec/**/*.rb'
+                      ]
   gem.require_paths = ['lib']
 
-  gem.add_dependency             'redcarpet', '~> 2.0'
+  gem.add_dependency             'redcarpet', '~> 3.4.0'
```
Comment (adamstegman) [inline redcarpet-confluence.gemspec]: Could you use `3.4` to make it less restrictive? This should in theory work on any 3.x version.

**Review by adamstegman — COMMENTED** [review]
Thank you @tomlockwood! I haven't used Confluence in years, so I'm glad you want to bring the project up to date.

It looks like the tests are still passing and the binary still works. I just left a couple minor comments but this looks good!

**Comment by tomlockwood** [comment]
Thanks heaps!

**Pull request merged**